types/k6/experimental/browser/element_handle.d.ts

```diff
@@ -0,0 +1,210 @@
+  isDisabled(): boolean;
+
+  /**
+   * Checks if the element is editable.
+   * @returns Whether the element is editable.
+   */
+  isEditable(): boolean;
+
+  /**
+   * Checks if the element is enabled.
+   * @returns Whether the element is enabled.
+   */
+  isEnabled(): boolean;
+
+  /**
+   * Checks if the element is hidden.
+   * @returns Whether the element is hidden.
+   */
+  isHidden(): boolean;
+
+  /**
+   * Checks if the element is visible.
+   * @returns Whether the element is visible.
+   */
+  isVisible(): boolean;
+
+  /**
+   * Returns the frame containing the given element.
+   * @returns The frame that contains the element handle.
+   */
+  ownerFrame(): Frame;
+
+  /**
+   * Focuses the element, and then uses `keyboard.down` and `keyboard.up`
+   * with each character in the `key` string.
+   * @param key Characters to type into the element.
+   * @param options Keyboard press options.
+   */
+  press(key: string, options?: KeyboardPressOptions): void;
+
+  /**
+   * This method scrolls element into view if needed, and then captures a
+   * screenshot of it.
+   * @param options Screenshot options.
+   * @returns An `ArrayBuffer` with the screenshot data.
+   */
+  screenshot(options?: ScreenshotOptions & TimeoutOptions): ArrayBuffer;
+
+  /**
+   * This method checks whether the element is actionable using provided options, and
+   * then tries to scroll it into view, unless it is completely visible.
+   * @param options Element handle options.
+   */
+  scrollIntoViewIfNeeded(options?: ElementHandleOptions): void;
+
+  /**
+   * Select one or more options of a `<select>` element which match the values.
+   * @param values Values of options to select.
+   * @param options Element handle options.
+   * @returns List of selected options.
+   */
+  selectOptions(values: string | ElementHandle | SelectOptionsObject | string[] | ElementHandle[] | SelectOptionsObject[], options?: ElementHandleOptions): string[];
+
+  /**
+   * Focuses the element and selects all its text content.
+   * @param options Element handle options.
+   */
+  selectText(options?: ElementHandleOptions): void;
+
+  /**
+   * Scrolls element into view if needed, and then uses `page.tapscreen` to tap in the center of the element
+   * or at the specified position.
+   * @param options Tap options.
+   */
+  tap(options?: MouseMoveOptions): void;
+
+  /**
+   * Returns the `node.textContent`.
+   * @returns The text content of the element.
+   */
+  textContent(): string;
+
+  /**
+   * Scrolls element into view, focuses element and types text.
+   * @param text Text to type into the element.
+   * @param options Typing options.
+   */
+  type(text: string, options?: KeyboardPressOptions): void;
+
+  /**
+   * Scrolls element into view, and if it's an input element of type
+   * checkbox that is already checked, clicks on it to mark it as unchecked.
+   * @param options Click options.
+   */
+  uncheck(options?: ElementClickOptions & StrictnessOptions): void;
```

Comment:
Looks like we have a bug in our code for this method, we shouldn't accept `StrictnessOptions` since `uncheck` should only work on the `ElementHandle` which is only associated with a single element.

```suggestion
  uncheck(options?: ElementClickOptions): void;
```

Reply:
I'd suggest that we keep `strict` here, since we agreed that type definitions should be reflecting the current implementation. Could you open a bug issue for this in `xk6-browser`? Once it's fixed, we can remove `StrictnessOptions` from the type definition.

Reply:
Actually, I'll file this bug myself, since I'm on it anyway :)